wwwroot/cgi-bin/lib/robots.pm

```diff
@@ -400,26 +404,26 @@
 'bingpreview',
 'MSIECrawler',
 'msnbot/',
-'msnbot\-media/',
+'msnbot-media/',
 'AdIdxBot/',
 'NOT[\x20]Googlebot/',
 'Googlebot/',
 'Google[\x20]Web[\x20]Preview',
-'Googlebot\-Image/',
-'Googlebot\-Mobile/',
-'google\-sitemaps',
-'Googlebot\-News',
-'Googlebot\-Video/',
-'AdsBot\-Google[\x20]\(',
-'AdsBot\-Google\-Mobile\-Apps',
+'Googlebot-Image/',
+'Googlebot-Mobile/',
+'google-sitemaps',
+'Googlebot-News',
+'Googlebot-Video/',
+'AdsBot-Google[\x20]\(',
+'AdsBot-Google-Mobile-Apps',
 'Mediapartners-Google',
-'Feedfetcher\-Google',
-'Google\-Adwords\-Instant',
+'Feedfetcher-Google',
+'Google-Adwords-Instant',
 'Firefox/1\.5',
 'Yahoo![\x20]Slurp[\x20]China',
 'Yahoo![\x20]Slurp',
 'Baiduspider/',
-'Baiduspider\-image',
+'Baiduspider-image',

```

Comment:
because the escaping is not necessary. The hyphen is only a special regex chararcter inside [ ]
Removing the hyphen where its not required lessens the load on regex and makes the files more readable.